drive/src/fee/removed_bytes_from_epochs_maps.rs

```diff
@@ -0,0 +1,91 @@
+use crate::error::fee::FeeError;
+use crate::error::Error;
+use bincode::Options;
+use costs::storage_cost::removal::Identifier;
+use intmap::IntMap;
+use serde::{Deserialize, Serialize};
+use std::collections::btree_map::{IntoIter, Iter};
+use std::collections::BTreeMap;
+
+/// Fee Result
+#[derive(Debug, Clone, Eq, PartialEq, Default, Serialize, Deserialize)]
+pub struct RemovedBytesFromIdentities(pub BTreeMap<Identifier, IntMap<u32>>);
+
+impl RemovedBytesFromIdentities {
```

Comment:
Structure and module have different names. The module's one makes more sense to me

Reply:
Fixed